package-lock.json

```diff
@@ -11677,12 +11677,14 @@
             "balanced-match": {
               "version": "1.0.0",
               "bundled": true,
-              "dev": true
+              "dev": true,
+              "optional": true
```

Comment:
Were these changes to package-lock.json intended?

Reply:
Reverted this, I'm working from home today so I needed to run an `npm install` which made these changes and they didn't need to be checked in